trento/xml/article_sap_trento.xml

```diff
@@ -669,6 +669,120 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
+      <listitem>
+        <para>cleanup:all: allows triggering housekeeping actions on hosts
+          where agents heartbeat is lost and SAP or HANA instances that are no
+          longer found</para>
+      </listitem>
+      <listitem>
+        <para>all:settings: grants changing capabilities on any system
+          settings under the <guimenu>Settings</guimenu> view</para>
+      </listitem>
+      <listitem>
+        <para>all:all: grants all the permissions above</para>
+      </listitem>
+    </itemizedlist>
+    <para>Using the described permissions, it is possible to create the following types of users:</para>
+    <itemizedlist>
+      <listitem>
+        <para>User managers: users with all:users permission</para>
+      </listitem>
+      <listitem>
+        <para>SAP administrator with Trento display-only access: users with
+          default permissions</para>
+      </listitem>
+      <listitem>
+        <para>SAP administrator with Trento configuration access: users with
+          all:checks_selection, all:tags and all:settings permissions</para>
+      </listitem>
+      <listitem>
+        <para>SAP administrator with Trento operation access: users with
+          all:check_execution and cleanup:all permissions.</para>
+      </listitem>
+    </itemizedlist>
+    <para>The default admin user created during the installation process is
+      granted all:all permissions and cannot be modified or deleted. Use it
+      only to create a first user manager. That is, a user with
+      all:users permissions who creates all the other required
+      users. Once a user with all:users permissions is created, the default
+      admin user must be regarded as a fall-back user to be used
+      only in case all other access to the console is lost. If the password
+      of the default admin user is lost, it can be reset by updating the
+      helm chart or the web component configuration, depending on which
+      deployment method was used to install Trento Server. </para>
+    <para>User passwords, including the default admin user password, must follow the rules below:</para>
+    <itemizedlist>
+      <listitem>
+        <para>Password must contain at least 8 characters</para>
+      </listitem>
+      <listitem>
+        <para>The same number or letter must not be repeated three or more times in a row (for
+          example: 111 or aaa)</para>
+      </listitem>
+      <listitem>
+        <para>Password must not contain four consecutive numbers or letters (for example:
+          1234, abcd or ABCD)</para>
+      </listitem>
+    </itemizedlist>
+    <para>The Create User and Edit User views provide a built-in generation
+      password action button that allows user managers to easily generate
+      secure and compliant passwords. The user manager must provide the user with
+      their password through an authorized secure channel. </para>
+    <para>A user can reset their password in the profile view. Here, they can
```

Comment:
Is "profile view" the name of the actual view of the user? In that case, check if the name should be capitalized.

Reply:
Yes, it is.
```suggestion
    <para>A user can reset their password in the <guimenu>Profile</guimenu> view. Here, they can
```